<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>linacPlcWidget</class>
 <widget class="TaurusWidget" name="linacPlcWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>312</width>
    <height>359</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="TaurusGroupBox" name="plcGroup">
      <property name="title">
       <string>PLC #</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout">
         <item row="3" column="1">
          <widget class="LinacLed" name="stateRead">
           <property name="minimumSize">
            <size>
             <width>25</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>25</width>
             <height>25</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="statusLabel">
           <property name="text">
            <string>Status</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="LinacLed" name="lockingRead">
             <property name="minimumSize">
              <size>
               <width>25</width>
               <height>25</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>25</width>
               <height>25</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="LinacValueCheckBox" name="lockingWrite">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>25</width>
               <height>20</height>
              </size>
             </property>
             <property name="model" stdset="0">
              <string/>
             </property>
             <property name="useParentModel" stdset="0">
              <bool>false</bool>
             </property>
             <property name="autoApply" stdset="0">
              <bool>true</bool>
             </property>
             <property name="forcedApply" stdset="0">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="lockingLabel">
           <property name="text">
            <string>Locking</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="stateLabel">
           <property name="text">
            <string>State</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="LinacLed" name="heartbeatRead">
           <property name="minimumSize">
            <size>
             <width>25</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>25</width>
             <height>25</height>
            </size>
           </property>
           <property name="alignment">
            <set>Qt::AlignBottom|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="heartbeatLabel">
           <property name="text">
            <string>Heartbeat</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="lockerLabel">
           <property name="text">
            <string>Locker</string>
           </property>
          </widget>
         </item>
         <item row="7" column="2">
          <widget class="TaurusLabel" name="forceWriteDB">
           <property name="bgRole">
            <string>none</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0" colspan="2">
          <widget class="TaurusCommandButton" name="RestoreReadDB">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="6" column="0" colspan="3">
          <widget class="TaurusCommandButton" name="ResetState">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="4" column="1" colspan="2">
          <widget class="TaurusLabel" name="statusRead">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="1" column="1" colspan="2">
          <widget class="TaurusLabel" name="lockerRead">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>25</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="TaurusGroupBox" name="instanceGroup">
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <layout class="QGridLayout" name="instanceLayout">
         <item row="1" column="0">
          <widget class="QLabel" name="instanceLocationLabel">
           <property name="text">
            <string>Instance location</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="TaurusCommandButton" name="moveLocal">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string>---</string>
           </property>
           <property name="Command">
            <string>MoveInstance</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" colspan="2">
          <widget class="TaurusLabel" name="instanceLocationRead">
           <property name="minimumSize">
            <size>
             <width>25</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>25</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="2">
          <widget class="LinacLed" name="instanceStateRead">
           <property name="minimumSize">
            <size>
             <width>25</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>25</width>
             <height>25</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="instanceStateLabel">
           <property name="text">
            <string>Instance state</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="TaurusCommandButton" name="moveRemote">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string>---</string>
           </property>
           <property name="Command">
            <string>MoveInstance</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="TaurusCommandButton" name="resetInstance">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TaurusWidget</class>
   <extends>QWidget</extends>
   <header>taurus.qt.qtgui.container</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TaurusGroupBox</class>
   <extends>QGroupBox</extends>
   <header>taurus.qt.qtgui.container</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TaurusLabel</class>
   <extends>QLabel</extends>
   <header>taurus.qt.qtgui.display</header>
  </customwidget>
  <customwidget>
   <class>TaurusCommandButton</class>
   <extends>QPushButton</extends>
   <header>taurus.qt.qtgui.button</header>
  </customwidget>
  <customwidget>
   <class>TaurusLed</class>
   <extends>QLed</extends>
   <header>taurus.qt.qtgui.display</header>
  </customwidget>
  <customwidget>
   <class>QLed</class>
   <extends>QPixmapWidget</extends>
   <header>taurus.qt.qtgui.display</header>
  </customwidget>
  <customwidget>
   <class>QPixmapWidget</class>
   <extends>QWidget</extends>
   <header>taurus.qt.qtgui.display</header>
  </customwidget>
  <customwidget>
   <class>LinacValueCheckBox</class>
   <extends>QCheckBox</extends>
   <header>ctli</header>
  </customwidget>
  <customwidget>
   <class>LinacLed</class>
   <extends>TaurusLed</extends>
   <header>ctli</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
